<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyRezervation</class>
 <widget class="QDialog" name="modifyRezervation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nazwa Filmu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="movieNameLE">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data i godzina</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="timeandDateCB"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ilość biletów</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ticketLE"/>
   </item>
   <item>
    <widget class="QTextBrowser" name="text"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Odwołaj rezerwację</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Wybierz miejsca</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
